ui/eslint.config.js

```diff
@@ -12,26 +12,22 @@ const config = [
       'no-shadow': 'off',
       'no-unused-vars': 'off',
       'prefer-destructuring': 'off',
-      'import/no-named-as-default': 'off',
-      'import/no-named-as-default-member': 'off',
       'jsx-a11y/anchor-is-valid': 'off',
-      'jsx-a11y/aria-role': 'off',
       'jsx-a11y/control-has-associated-label': 'off',
       'promise/always-return': 'off',
       'react/destructuring-assignment': 'off',
       'react/jsx-no-leaked-render': 'off',
       'react/no-multi-comp': 'off',
       'react/no-unused-class-component-methods': 'off',
-      'react/no-unstable-nested-components': 'off',
```

Comment:
Loosened in v1.0.4 to allow the use of the render prop pattern: https://gitlab.esrf.fr/ui/eslint-config/-/commit/58f7cdc7d513a04e184c774d5dd2083cf2d759b3